MainPage.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+var React = require('react-native');
```

Comment:
prefer `import React, { Component } from 'react-native'` instead of `require`.

Require is good for other types of imports